src/util/util_bit.h

```diff
@@ -178,8 +182,15 @@ namespace dxvk::bit {
   inline uint32_t lzcnt(uint64_t n) {
     #if defined(_MSC_VER) && !defined(__AVX2__)
     unsigned long bsr;
-    _BitScanReverse(&bsr, n);
-    return 31-bsr;
+    #if defined(DXVK_ARCH_X86_64)
+    _BitScanReverse64(&bsr, n);
+    #else
```

Comment:
We already have a fallback path below that uses the 32-bit implementation, no need to duplicate that here.